### ISSUE TITLE
Is there anything that can be made/built/collaged/etc. from actual old books?

### ISSUE THREAD

**Issue by bjornmeansbear** [GD399-OSD/library-identity#7]
related to #6 

Stacks of books to make giant letterforms?
torn up pages turned into posters/collages/etc?
what else could we come up with?

**Comment by gzhang95** [comment]
Book sculpture?  We can turn the old book into a model of the internal details of Decker library, or we can make them into some MICA'S landmark buildings(Main building, Brown, etc).  It may take a lot of time, but the result will be very impressive.
https://pre00.deviantart.net/30ab/th/pre/i/2015/106/9/0/mary_poppins_book_sculpture_by_malenavalcarcel-d7vek3s.jpg
http://www.jamiebhannigan.com/wp-content/uploads/2015/06/Mr.-Mrs.jpg
http://strictlypaper.com/blog/wp-content/uploads/2013/08/liu-wei-foreign-640x854.jpeg

**Comment by bjornmeansbear** [comment]
@gzhang95 yeah, cool. Maybe there is some way to think about that typographic one as it could pertain to a library version of the MICA lockup or just the logoless logo?

Great finds!
